extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/GrantTokens.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.oidc.client;
+
+/**
+ * Access and Refresh tokens returned from a token grant request
+ */
+public class GrantTokens {
```

Comment:
You probably have a reason for that ... Wouldn't be better `Tokens` ?

Reply:
I did not like `GrantTokens` first but then somehow got used to it :-), no particular reason, just wanted to say the tokens is product of the grant request, but just `Tokens` is simpler I guess

Reply:
done